src/add-remove.js

```diff
@@ -0,0 +1,110 @@
+  static initialize() {
+    const todoTask = new Task();
+    return todoTask;
+  }
+
+  constructor() {
+    this.divLi = document.querySelector('.divLi');
+    this.addBtn = document.querySelector('.submitButton');
+    this.taskInput = document.querySelector('.taskInput');
+    this.arr = [];
+    this.loadTask();
+    this.addBtn.addEventListener('click', () => this.addTask(this.taskInput.value, this.arr.length));
+    this.taskInput.addEventListener('keydown', (event) => {
+      if (event.key === 'Enter') {
+        this.addTask(this.taskInput.value, this.arr.length);
+      }
+    });
+  }
+
+  saveTask = () => {
+    localStorage.setItem('task', JSON.stringify(this.arr));
+  }
+
+  displayTask = () => {
+    this.divLi.innerHTML = '';
+    this.arr.forEach((task, index) => {
+      const elementLi = document.createElement('li');
+      const elementDiv = document.createElement('div');
+      const elementDiv2 = document.createElement('div');
+      const elementInput = document.createElement('input');
+      const elementLabel = document.createElement('input');
+      const elementButton = document.createElement('button');
+      const elementDeleteButton = document.createElement('button');
+      elementButton.classList.add('threeDots');
+      elementDeleteButton.classList.add('elementDeleteButton');
+      elementDiv.classList.add('elementDiv1');
+      elementDiv2.classList.add('elementDiv2');
+      elementLabel.classList.add('taskLabel');
+
+      this.divLi.appendChild(elementLi);
+      elementLi.appendChild(elementDiv);
+      elementDiv.appendChild(elementInput);
+      elementDiv.appendChild(elementLabel);
+      elementLi.appendChild(elementDiv2);
+      elementDiv2.appendChild(elementDeleteButton);
+      elementDiv2.appendChild(elementButton);
+      elementInput.type = 'checkbox';
+      elementLabel.value = `${task.taskToDo}`;
+      elementDeleteButton.textContent = 'X';
+      elementButton.textContent = '...';
+
+      elementDeleteButton.addEventListener('click', () => {
+        this.removeTask(index);
+      });
+
+      elementLabel.addEventListener('focusin', () => {
+        elementLi.style.boxShadow = '0 0 10px rgba(5, 110, 200, 0.8)';
+      });
+      elementLabel.addEventListener('focusout', () => {
+        elementLi.style.border = '1px solid rgba(0, 0, 0, 0.15)';
+        elementLi.style.boxShadow = 'none';
+        task.taskToDo = elementLabel.value;
+        this.saveTask();
+      });
+    });
+    this.taskInput.placeholder = 'Add to your list...';
+    this.taskInput.classList.remove('taskInputERROR');
+  }
+
+  addTask = (newValue) => {
+    const taskToDo = newValue;
+    const solved = false;
+    const taskNumber = this.arr.length;
+
+    if (taskToDo === '') {
+      this.taskInput.placeholder = 'FILL THIS FIELD';
+      this.taskInput.classList.add('taskInputERROR');
+      return;
+    }
+
+    const task = {
+      taskToDo,
+      solved,
+      taskNumber,
+    };
```

Comment:
- [ ] As per the previous reviewer's suggestion the `taskNumber/index` should be start from `1`. But I can see the `taskNumber/index` start in your to-do-list from `0`. The serial should be `1`,`2`,`3`, But here I can see `0`,`1`,`2`. Kindly fix this issue.
- ![image](https://github.com/Zven94/to-do-list/assets/60083437/4a90e499-a209-4c9b-8bf7-d869bd70153f)

Reply:
Thank you again Tufahel! I already solve it i'm requesting it again 💪